tests/integration/mixer/outboundtrafficpolicy/v2/traffic_registry_only_test.go

```diff
@@ -67,17 +65,19 @@ func TestOutboundTrafficPolicy_RegistryOnly_TelemetryV2(t *testing.T) {
 			Name:     "TCP",
 			PortName: "tcp",
 			Expected: outboundtrafficpolicy.Expected{
-				Metric:          "istio_tcp_connections_closed_total",
-				PromQueryFormat: `sum(istio_tcp_connections_closed_total{reporter="source",destination_service_name="BlackHoleCluster",source_workload="client-v1"})`,
+				// TODO(https://github.com/istio/istio/issues/22717) re-enable TCP
```

Comment:
From the logs in the bug, it does not look like they failed for Passthrough cluster and not Blackhole cluster?

Reply:
John disabled them for mixer telemetry https://github.com/istio/istio/pull/22707/files#diff-9ca8ef00ed0f610d93a890f207e8dd96R66 due to flakiness. When I do my local testing I'm not able to replicate the flakiness. I can revert it and we can put up a PR if it is flaky.

Reply:
Sounds good !